metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1653,6 +1658,26 @@ class MetalsLanguageServer(
       .foreach(_ => languageClient.refreshModel())
   }
 
+  private def checkRunningBloopVersion(bspServerVersion: String) = {
```

Comment:
nit: should we move this into a separate file?

Reply:
I think we should do it as a larger refactor of the file. I can see a number of similar methods that might be put there.